### DISCUSSION TITLE
20220204 Static, white noise and distortion

### DISCUSSION THREAD

**Discussion by sibezwo** [rern/rAudio#801] [Bug Reports]
Hello Rern

The latest update is causing me intermittent static and after a while loud white noise and distortion. The fresh current image i20220116 works fine until I do the update.

The same problem kept me from switching to the 64bit image for a long time. But so far it has been stable. I could easily listen to web radio continuously for 48 hours.

RPi3B with USB DAC/AMP (SMSL Q5Pro)
Volume Control: MPD Software

Thomas

**Comment by R101G** [comment]
@sibezwo 

You don't say whether this is affected by the volume setting, or whether it only affects radio sources, or whether you are using wifi or Ethernet.

It could be that the update has changed a setting (Bluetooth/wifi enabled?) or has increased the load on the Pi. I find my old 3B struggles with the 64 bit image and makes noises when the GUI is updated. If it all looks OK I would try the 32 bit image again.

**Comment by ceejayemm** [comment]
A mix of RPi 3A, 3B and 3B+ all on update 20220204 64 bit with a mix of different DACs, one 3B wired, all the rest are wireless, music library and shared data via NFS share, mostly default settings. Listening to wireless web radio (UK BBC stations) during day and albums at other times. No apparent white noise, distortion or noises when GUI is updated noted with any of these systems.

Chris

**Comment by sibezwo** [comment]
After a reboot, rAudio starts playing the stream. Statics can occasionally be heard. My Pixel 4 is not connected to the Pi, neither via Chrome nor via Bluetooth. I don't do anything. At some point I hear loud white noise and loud distorted music fragments until I stop the stream.

Increasing the audio and/or output buffer did not help. Bluetooth on or off doesn't matter. Neither does Chromium Browser on Pi. I tried several times to determine the problem. But without success, because I did the update immediately after writing the image.

With the fresh image, I've been listening non-stop for 45 hours now with no problems. As you can see on the screenshots.

But not being able to update is not a satisfactory solution. That's why I thought about ordering an IQaudio DigiAMP+.

Thomas

<img src="https://user-images.githubusercontent.com/78496250/153701698-fd9e958d-ad70-41b3-86a0-d941d360dd19.png" width=300>

<img src="https://user-images.githubusercontent.com/78496250/153701743-4a29cd0e-9869-48c8-95bc-5734e29e4dd4.png" width=300>

**Comment by sibezwo** [comment]
Sorry, I forgot to mention what my setup looks like: 

Router - Wi-Fi - RPi3B - USB - SMSL Q5Pro

Nothing else is connected to the Pi, not even a flash drive. Sometimes I connect my Pixel 4 via Bluetooth to listen to a few songs on YouTube. But usually I have Bluetooth turned off.

Thomas

**Comment by R101G** [comment]
@sibezwo 
With such a configurable system, the chance of another user having exactly the same setup as you is small. I gave up on 64 bit with the original 3B, but @ceejayemm has no problems. It would be interesting to see the system information after the update, as well as before.

I would still suggest trying the 32 bit image before making any hardware changes. Obviously, there is nothing wrong with the update, or it would affect everyone.

EDIT: I just noticed you are not using the latest image of rAudio. You might want to try that first, without using backup/restore for the settings.
